download/download.go

```diff
@@ -83,6 +85,10 @@ func (d *Download) Get() (err error) {
 		return
 	}
 
+	frontedURL := *req.URL
+	frontedURL.Host = "d2yl1zps97e5mx.cloudfront.net"
+	proxied.PrepareForFronting(req, frontedURL.String())
+
```

Comment:
Just based on that existing comment below it looks like we might want to do this inside of the below `RoundTripper` definition @atavism? Seems odd but safer just based on the comment about redirects.